modules/virt-creating-nad-localnet-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-localnet-console_{context}"]
+= Creating a NAD for localnet topology using the web console
+
+You can create a network attachment definition (NAD) to connect workloads to a physical network by using the {product-title} web console.
+
+.Prerequisites
+* Use nmstate to configure a localnet identifier.
+
+.Procedure
+
+. Navigate to *Networking* -> *NetworkAttachmentDefinitions* in the web console.
+
+. Click *Create Network Attachment Definition*. The network attachment definition must be in the same namespace as the pod or virtual machine using it.
+
+. Enter a unique *Name* and optional *Description*.
```

Comment:
I'm curious: this description attribute is only for the UI right ? There's nothing in the NAD to store this afaik.

Reply:
@phoracek See question above.

Reply:
AFAIK UI stores the description in the NAD's annotation